ADAL/src/ADAuthenticationErrorConverter.m

```diff
@@ -48,8 +49,10 @@ + (void)initialize
     s_errorCodeMapping = @{
                            //sample format is like @"MSIDErrorDomain|-10000":@"-20000"
                            @"MSIDErrorDomain|-51001":@"212",
-                           @"MSIDErrorDomain|-51002":@"101"
+                           @"MSIDErrorDomain|-51301":@"101"
```

Comment:
We should probably change this to something that's not relying on string with the hardcoded numbers, they should be using the enum values directly.